### PR TITLE
DOCKER-129 Improve the process of locally built image

### PR DESCRIPTION
**Pull request by Akos-Kreutz**
For example:

]$ ./build_local_image.sh ~/bundles/ dxp snapshot no_warm_up no_test_image

Builds the image in less than 30 seconds

### CODE REVIEW

File: _common.sh [hunk 1]
```diff
@@ -192,15 +192,18 @@ function prepare_tomcat {
...
 
 	rm -fr "${TEMP_DIR}"/liferay/logs/*
 	rm -fr "${TEMP_DIR}"/liferay/tomcat/logs/*
 }
 
-function remove_temp_dockerfile_target_platform {
-	sed -i 's/${TARGETARCH}'/$(get_current_arch)/ "${TEMP_DIR}"/Dockerfile
-	sed -i 's/--platform=${TARGETPLATFORM} //g' "${TEMP_DIR}"/Dockerfile
+function remove_temp_dockerfile_platform_variable {
+	sed -i'' 's/${TARGETARCH}'/$(get_current_arch)/ "${TEMP_DIR}"/Dockerfile
```
Comment (gamerson) [inline _common.sh:205]:
```suggestion
	sed -i'.bak' 's/${TARGETARCH}'/$(get_current_arch)/ "${TEMP_DIR}"/Dockerfile
```

Comment (gamerson) [reply _common.sh:205]: For MacOS, you must provide a backup extension when `sed` is doing inline editing.  The `-i'.bak'` is compatible with GNU/linux and BSD/MacOS.

**Review by gamerson — CHANGES_REQUESTED** [review]

File: _common.sh [hunk 2]
```diff
@@ -192,15 +192,18 @@ function prepare_tomcat {
...
 	rm -fr "${TEMP_DIR}"/liferay/logs/*
 	rm -fr "${TEMP_DIR}"/liferay/tomcat/logs/*
 }
 
-function remove_temp_dockerfile_target_platform {
-	sed -i 's/${TARGETARCH}'/$(get_current_arch)/ "${TEMP_DIR}"/Dockerfile
-	sed -i 's/--platform=${TARGETPLATFORM} //g' "${TEMP_DIR}"/Dockerfile
+function remove_temp_dockerfile_platform_variable {
+	sed -i'.bak' 's/${TARGETARCH}'/$(get_current_arch)/ "${TEMP_DIR}"/Dockerfile
+	sed -i'' 's/--platform=${TARGETPLATFORM} //g' "${TEMP_DIR}"/Dockerfile
```
Comment (gamerson) [inline _common.sh:206]:
```suggestion
	sed -i'.bak' 's/--platform=${TARGETPLATFORM} //g' "${TEMP_DIR}"/Dockerfile
```
There is another one :)

**Comment by brianchandotcom** [comment]
@gamerson @Akos-Kreutz @rotty3000 this is in!

**Comment by gamerson** [comment]
thanks @brianchandotcom  and @Akos-Kreutz   this helps a lot.

**Comment by rotty3000** [comment]
:tada:

**Pull request closed without merging**